server/settings/workfile_builder.py

```diff
@@ -10,16 +10,18 @@ class CustomBuilderTemplate(BaseSettingsModel):
         default_factory=list,
         title="Task types",
     )
-    template_path: MultiplatformPathModel = SettingsField(
+    path: MultiplatformPathModel = SettingsField(
         default_factory=MultiplatformPathModel
     )
 
 
 class WorkfileBuilderPlugin(BaseSettingsModel):
+    """Legacy use of simple workfile template based on Task type"""
     _title = "Workfile Builder"
     create_first_version: bool = SettingsField(
         False,
-        title="Create first workfile"
+        title="Create first workfile",
+        description="Must be enabled to copy template and create first workfile"
```

Comment:
This description to me is confusing - why must "Create first workfile" be enabled 'to copy template'? Doesn't this just mean "Save first workfile with the built template on first run if no existing workfile exists."?

Reply:
Yeah, I just wanted highlight, that the toggle must be enabled, not really sure why it is there. `Create first workfile` is a process of `copy template and create first workfile (from template)`. But yours description is better.